mds/views.py

```diff
@@ -109,38 +110,63 @@ def perform_update(self, serializer):
         serializer.save()
 
 
+class DeviceLimitOffsetPagination(LimitOffsetPagination):
+    default_limit = 500
```

Comment:
Any reason to use LimitOffset and not PageNumber or Cursor ?
Note Cursor pagination does not suffer from the limitations of other paginations - this may not be relevant here, however.

Reply:
No particular reason besides personal preference, it may not be the optimal solution but I think this is good enough for starters.